assets/js/theme/common/product-details.js

```diff
@@ -233,6 +238,10 @@ export default class ProductDetails extends ProductDetailsBase {
         });
     }
 
+    checkIsInQuickView($element) {
```

Comment:
what's your thought to use `isElementInsideQuickView` as method's name ?

Reply:
Honestly I think it is little bit excess, because function has $element as a parameter and its adding the same context

Reply:
checkIsQuickViewChild

Reply:
&#9989;